### ISSUE TITLE
Feature request : Falstad Circuit simulator android

### ISSUE THREAD

**Issue by Awen2345** [sharpie7/circuitjs1#633]
Hi, is there an android version of Falstad Circuit simulator?

**Comment by Jeppe53** [comment]
> Hi, is there an android version of Falstad Circuit simulator?

Yes - You just visit the website from a browser on your phone :)

**Comment by dreng** [comment]
Unfortunately the browser version is unusable on phones or tablets. Just try it yourself and you will find out in seconds.

I would appreciate an android version with a customized UI very much as well.

**Comment by Noitarud** [comment]
@pfalstad maybe this link will help you accomplish the feat:
https://www.codeproject.com/articles/269318/an-adventure-in-porting-a-java-applet-to-android-w

(Also requesting for Fdroid)
